crates/rspack_core/src/dependency/code_generatable.rs

```diff
@@ -257,70 +257,70 @@ pub fn javascript_path_to(
 
 #[macro_export]
 macro_rules! create_css_visitor {
-  (exact $ast_path:expr, $name:ident($arg:ident: &mut $ty:ident) $b:block) => {
-      $crate::create_css_visitor!(__ $ast_path.to_vec(), $name($arg: &mut $ty) $b)
-  };
-  ($ast_path:expr, $name:ident($arg:ident: &mut $ty:ident) $b:block) => {
-      $crate::create_css_visitor!(__ $crate::css_path_to(&$ast_path, |n| {
-          matches!(n, swc_core::css::visit::AstParentKind::$ty(_))
-      }), $name($arg: &mut $ty) $b)
-  };
-  (__ $ast_path:expr, $name:ident($arg:ident: &mut $ty:ident) $b:block) => {{
-      struct Visitor<T: Fn(&mut swc_core::css::ast::$ty) + Send + Sync> {
-          $name: T,
+  (visit_mut_stylesheet($arg:ident: &mut Stylesheet) $b:block) => {{
```

Comment:
The macro broke, fixed by reordering this to the top.

Reply:
cc @h-a-n-a PTAL

Reply:
looks good to me